README.md

```diff
@@ -47,8 +47,10 @@ Repo for automation build, test etc.
            python3 tools/install_venv.py
             ```
     - Ubuntu 18.04
-         - Install required packages
+         - Install required packages including openstack client version (min 4.0.2 is required).
            ```
+           add-apt-repository cloud-archive:ussuri
+           apt install/upgrade python3-openstackclient           
```

Comment:
You need to provide the exact command here. Not like `install/upgrade`

Reply:
done